dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/QueueServiceImpl.java

```diff
@@ -160,7 +160,7 @@ public Map<String, Object> createQueue(User loginUser, String queue, String queu
         queueMapper.insert(queueObj);
         result.put(Constants.DATA_LIST, queueObj);
         putMsg(result, Status.SUCCESS);
-
+        resourcePermissionCheckService.postHandle(AuthorizationType.QUEUE, loginUser.getId(), Collections.singletonList(queueObj.getId()), logger);
```

Comment:
I have some question:

1. Why to check resource permission of the new created queue here? 
2. If check fail, should the db transaction rollback?

Reply:
This is a post action, which is extended to save the data permission relationship in ws. When an exception occurs, a Runtime exception will be thrown and rolled back.

Reply:
But I don't see the @Transactional of this method, is it right?

Reply:
Yes I still have this question, do we need to rollback here?

Reply:
I think it is needed, including other methods involving data manipulation. This can be used as an optimization item to do an overall transaction investigation later.

Reply:
updated checkList. @caishunfeng @ruanwenjun